### PR TITLE
Password overlay header update

### PR DESCRIPTION
**Pull request by raitalharehman**
In password overlay header title add class overlay__header-text and changed the tag from `p` to `div`
just like all other overlays

### CODE REVIEW

File: src/components/OverlayManager/Password/index.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const Password: React.FC<{ overlay: OverlayContextInterface }> = ({
     <div className="password-reset">
       <Online>
         <div className="overlay__header">
-          <p>Reset your password</p>
+          <div className="overlay__header-text">Reset your password</div>
```
Comment (piotrgrundas) [inline src/components/OverlayManager/Password/index.tsx]: Div should not have directly text in it.
Also please clean commits.

Comment (piotrgrundas) [inline src/components/OverlayManager/Password/index.tsx]: Div should not have directly text in it.
Also please clean commits.

File: README.md [hunk 2]
```diff
@@ -31,6 +31,8 @@ These instructions will get you a copy of the project up and running on your loc
 - A running instance of Saleor.
 
     To run the storefront, you have to set the `BACKEND_URL` environment to point to the Saleor instance. If you are running Saleor locally with the default settings, set `BACKEND_URL` to: `http://localhost:8000/`.
+    <br>**For Quick Demo** use `BACKEND_URL` = `https://demo.getsaleor.com` <br>
```
Comment (NyanKiyoshi) [inline README.md]: Could you create a new branch for your changes and remove the edits of the readme? Same thing for #270.

Comment (raitalharehman) [reply README.md]: okay

File: src/components/OverlayManager/Password/index.tsx (hunk 1 above)
Comment (raitalharehman) [reply src/components/OverlayManager/Password/index.tsx]: @piotrgrundas in many components `Div` have directly text in it . Should i change them all?

**Pull request closed without merging**